### ISSUE TITLE
9.0 PlayerNames.lua and Editbox.lua errors

### ISSUE THREAD

**Issue by Nillx** [sylvanaar/prat-3-0#141]
```
19x Prat-3.0-3.8.26\modules\PlayerNames.lua:1357: attempt to call global 'GuildRoster' (a nil value)
[string "@Prat-3.0-3.8.26\modules\PlayerNames.lua"]:1357: in function `?'
[string "@Bagnon\common\Wildpants\libs\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:119: in function <...nts\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:119>
[string "=[C]"]: ?
[string "@Bagnon\common\Wildpants\libs\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:29: in function <...nts\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
[string "@Bagnon\common\Wildpants\libs\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:64: in function `Fire'
[string "@Bagnon\common\Wildpants\libs\AceEvent-3.0\AceEvent-3.0-4.lua"]:120: in function <...\common\Wildpants\libs\AceEvent-3.0\AceEvent-3.0.lua:119>
```

```
1x Prat-3.0-3.8.26\modules\PlayerNames.lua:1155: attempt to call global 'GuildRoster' (a nil value)
[string "@Prat-3.0-3.8.26\modules\PlayerNames.lua"]:1155: in function `OnModuleEnable'
[string "@Prat-3.0-3.8.26\addon\modules.lua"]:189: in function <Prat-3.0\addon\modules.lua:178>
[string "=[C]"]: ?
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <Bartender4\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function `EnableAddon'
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:540: in function `EnableAddon'
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:630: in function <Bartender4\libs\AceAddon-3.0\AceAddon-3.0.lua:615>
```

```
1x Prat-3.0-3.8.26\modules\Editbox.lua:825: attempt to call method 'SetBackdrop' (a nil value)
[string "@Prat-3.0-3.8.26\modules\Editbox.lua"]:825: in function `SetBackdrop'
[string "@Prat-3.0-3.8.26\modules\Editbox.lua"]:753: in function `OnModuleEnable'
[string "@Prat-3.0-3.8.26\addon\modules.lua"]:189: in function <Prat-3.0\addon\modules.lua:178>
[string "=[C]"]: ?
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <Bartender4\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function `EnableAddon'
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:540: in function `EnableAddon'
[string "@Bartender4\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:630: in function <Bartender4\libs\AceAddon-3.0\AceAddon-3.0.lua:615>
```

**Comment by Vashidu** [comment]
#133 fixes the editbox, and then #142 should fix the others.

**Comment by Kream** [comment]
Confirmed @Vashidu 's #142 fixes the errors. works fine.
